pkg/logs/status/builder.go

```diff
@@ -103,6 +111,22 @@ func (b *Builder) getIntegrations() []Integration {
 	return integrations
 }
 
+// getTailers returns all the information about the logs integrations.
+func (b *Builder) getTailers() []Tailer {
+	var tailers []Tailer
+	for _, tailer := range b.tailers.All() {
+
+		info := tailer.GetInfo().Rendered()
+
+		tailers = append(tailers, Tailer{
+			Id:   tailer.GetId(),
+			Type: tailer.GetType(),
+			Info: info,
+		})
+	}
```

Comment:
Why not initialize the slice with the correct size directly ? (we know exactly how many elements will be in the slice)